spec/serializers/post_serializer_spec.rb

```diff
@@ -39,4 +40,10 @@
     serializer = PostSerializer.new(post, scope: guardian)
     expect(serializer.as_json[:post][:assignment_note]).to eq("tomtom best")
   end
+
+  it "includes status in serializer" do
```

Comment:
Would be good to add a spec here that checks that status is not included when `enable_assign_status` is not enabled.